frontend/src/views/Overview/components/TOverviewErrorView.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="info">
+    <div class="info__wrapper">
+      <t-icon class="info__icon" :icon="iconName" />
+      <h3 class="info__title">
+        {{ title }}
+      </h3>
+      <p class="info__subtitle">{{ subtitle }}</p>
+      <router-link to="/dashboard">
+        <t-button class="info__button" type="primary"
+          >Back to Dashboard</t-button
+        >
+      </router-link>
+    </div>
+  </div>
+</template>
+
+<script>
+import TIcon from "../../../components/common/Icon/TIcon.vue";
+import TButton from "../../../components/common/Button/TButton.vue";
```

Comment:
Do you know if it's possible to specify import paths relative to root in typescript somehow?
Like using `@/components` or something.